### DISCUSSION TITLE
Send recovery alerts

### DISCUSSION THREAD

**Discussion by Chris0408** [jertel/elastalert2#357] [Q&A] [answered]
when the problem that triggered the alert has recovered,Whether to support sending a "resolve" message ?

**Comment by nsano-rururu** [comment]
Not Support

**Reply by Chris0408** [reply]
I got it，thanks for your reply

**Comment by jertel** [comment, marked as answer]
See #115 for a method to accomplish this using a second rule.

**Reply by Chris0408** [reply]
ok，thank you
